### ISSUE TITLE
Waiting for confirmation after changing my cellphone

### ISSUE THREAD

**Issue by mauded5588** [bitpay/copay#6554]
#I transfered my wallet from android to ios and I'm waiting for the confirmation of a transaction that was already approved on my other device.  The transaction had been done over 5 months ago. Is it normal that it needs more confirmations? What can I do? HELP

**Comment by dabura667** [comment]
clear your cache and rescan

**Comment by mauded5588** [comment]
I never scaned the barcode to transfer my money I just open it on a new device. I can see on insight that there is over 23000 confirmations.

**Comment by mauded5588** [comment]
And if I go back to my previous cellphone, the amount is now waiting for a confirmation as if it has less than one confirmation, so I can't send it from my previous device.

**Comment by mauded5588** [comment]
It's also maybe due to the fact that I sent today 1$ to someone today just to do a test and this transaction is not confirmed yet. Can it affect a transfer that was done months ago?

**Comment by mauded5588** [comment]
Maybe it's because the fees of the transaction (0,50$) are comparable to those of the amount to be transfered?

**Comment by mauded5588** [comment]
Can I cancel this transaction because it was not confirmed yet and it seems to cause a problem?

**Comment by dabura667** [comment]
Did you clear your cache and rescan like I said?

**Comment by mauded5588** [comment]
I never scanned the barcode to transfer my money I just open it on a new device and now, on the other device, there is the same error, so I think that it's because of the small amount that I transferred. Do you think that this transaction will be completed soon even if it's a small fee?

**Comment by mauded5588** [comment]
I'm pretty sure that the problem is caused because I tranferred a too small amount to do a test (only 1$). I saw this error message : 
This transaction amount is too small compared to current Bitcoin network fees. Spending these funds will need a Bitcoin network fee cost comparable to the funds itself.

What should I do now, wait? or can I cancel this transaction? How?

I've been waiting for 7h for this to be approved and nothing happened...
